tests/v2_validation/cattlevalidationtest/core/test_rancher_cli.py

```diff
@@ -0,0 +1,620 @@
+
+    container_list = get_service_container_list(admin_client, service)
+    assert len(container_list) == 2
+    for container in container_list:
+        assert container.state == "running"
+
+    command = "restart " + stack_name + "/" + service.name
+    cli_response = execute_rancher_cli(client, stack_name, command,
+                                       service.id, docker_compose=None,
+                                       rancher_compose=None)
+    service = client.wait_success(service, 300)
+    if service.id in cli_response:
+        assert True
+
+    container_list = get_service_container_list(admin_client, service)
+    assert len(container_list) == 2
+    for container in container_list:
+        assert container.state == "running"
+        assert container.startCount == 2
+
+    delete_all(client, [stack])
+
+
+def test_cli_create_restart_service_batch_interval(admin_client, client,
+                                                   rancher_cli_container):
+
+    # This method restarts the service given batch-size and interval
+
+    stack_name = random_str().replace("-", "")
+    launch_rancher_cli_from_file(
+        client, RCLICOMMANDS_SUBDIR, stack_name,
+        "up -d", "Creating stack", "dc5.yml", "rc5.yml")
+
+    stack, service = get_env_service_by_name(client, stack_name, "rtest5")
+
+    # Confirm service is active and the containers are running
+    assert service.state == "active"
+    assert service.scale == 4
+    assert service.name == "rtest5"
+
+    check_config_for_service(admin_client, service, {"rtest5": "value5"}, 1)
+
+    container_list = get_service_container_list(admin_client, service)
+    assert len(container_list) == 4
+    for container in container_list:
+        assert container.state == "running"
+
+    command = "restart --type service --batch-size 2 --interval 1000 " \
+              + stack_name + "/" + service.name
+    cli_response = execute_rancher_cli(client, stack_name, command,
+                                       service.id, docker_compose=None,
+                                       rancher_compose=None)
+    service = client.wait_success(service)
+    if service.id in cli_response:
+        assert True
+
+    container_list = get_service_container_list(admin_client, service)
+    assert len(container_list) == 4
+    for container in container_list:
+        assert container.state == "running"
+        assert container.startCount == 2
+
+    delete_all(client, [stack])
+
+
+def test_cli_restart_container(admin_client, client,
+                               rancher_cli_container):
+
+    # This method deletes a standalone container
+
+    stack_name = random_str().replace("-", "")
+    container = []
+    container = client.create_container(name="test_cont",
+                                        networkMode=MANAGED_NETWORK,
+                                        imageUuid=TEST_IMAGE_UUID)
+
+    container = client.wait_success(container, 60)
```

Comment:
Included test for restarting containers that are part of a service. Test for stopping containers that  are part of a service will be done in the next checkin.